spec/spotify/artist_spec.cr

```diff
@@ -42,6 +42,7 @@ describe Spotify::Artist do
     it "populates the genres" do
       expect(artist.genres.size).to eq(2)
       expect(artist.genres).to be_a(Array(String))
+      expect(artist.genres[0]).to eq("blues-rock")
```

Comment:
Okay @marceloboeira.
I added one more test in the file as requested.
Thank you for the warning. :smile: